cypress/integration/pages/mediaAssetPage/testsForCanonicalOnly.js

```diff
@@ -32,6 +33,30 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
         },
       );
     });
+    it('should render an SMP player that is ready to play media', () => {
+      cy.window().then(win => {
+        const media =
```

Comment:
Do we also want to check for live streams?  The scenario `should render a media player` above checks for a live stream.  Perhaps this is something we want to do for the next set of services for MAPs.

Reply:
It checks for clips which have the 'video' type and live streams which have the 'version' type, so would fail if players on either of those don't show, or if there is no media for either of those.

Reply:
Do we have or need a test for audio?

Reply:
'video' is actually the type 'media ' (transformed somewhere in the process), and works for video and audio.